.github/workflows/integration-tests.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.8"
+          python-version: "3.9"
```

Comment:
`python-version: "3.8"` is still okay? 
https://github.com/ansible/ansible-rulebook/commit/79892c2d1c168de6f9ae1842fcb940606086a0b4

Reply:
yeah, unmade. thanks @konstruktoid